hooks/tk-maya/render_media.py

```diff
@@ -48,21 +49,30 @@ def render(
         # For more informations about the playblast API,
         # https://help.autodesk.com/view/MAYAUL/2020/ENU/?guid=__CommandsPython_playblast_html
 
+        is_windows = sys.platform == "win32"
```

Comment:
Can't wait for the python 3 branch to be merged to core and be able to do `sgtk.util.is_windows()/is_linux()/is_macos()`